modules/redpanda/redpanda_test.go

```diff
@@ -698,3 +701,19 @@ func TestRedpandaBootstrapConfig(t *testing.T) {
 		require.False(t, needsRestart)
 	}
 }
+
+func detectContainerHostAddress(t *testing.T, image string, ctx context.Context) string {
+	c, err := testcontainers.GenericContainer(ctx, testcontainers.GenericContainerRequest{
+		ContainerRequest: testcontainers.ContainerRequest{
+			Image: image,
+		},
+		Started: false,
+	})
+	require.NoErrorf(t, err, "failed to create stopped container")
```

Comment:
suggestion: just use NoError, we don't care what type it is just that it isn't an error.

More below...

Reply:
Done